templates/vrrp_instance.erb

```diff
@@ -31,6 +31,9 @@ vrrp_instance <%= @_name %> {
   <%- if @vmac_xmit_base -%>
   vmac_xmit_base
   <%- end -%>
+  <%- if @native_ipv6 -%>
```

Comment:
take a look at some lines above. Your codeblock is within the `use_vmac` condition. I am not sure if this is on purpose or not (I assume not).

Reply:
Ah, right. An excellent example of why tests are a good thing to have. :) Should be fixed in 5750127.